Sources/SwiftASN1/Basic ASN1 Types/GeneralizedTime.swift

```diff
@@ -116,6 +134,7 @@ public struct GeneralizedTime: DERImplicitlyTaggable, BERImplicitlyTaggable, Has
     @usableFromInline var _minutes: Int
     @usableFromInline var _seconds: Int
     @usableFromInline var _fractionalSeconds: Double
```

Comment:
can we document that `_fractionalSeconds` is now a cached value and that `_rawFractionalSeconds` is the source of truth?

Reply:
So, the existing `public init` takes `fractionalSeconds` and now computes the `rawFractionalSeconds` property (for a uniform property set as with the new `internal init`). In that case, `_fractionalSeconds` is not a cached value and `_rawFractionalSeconds` is not the source of truth.

Reply:
If this is the intention that this needs to be used throughout the implementation. Serialization would need to be aware of this and use  `_fractionalSeconds` instead of `_rawFractionalSeconds`. 

However, I think that is not what we want. The init that takes `fractionalSeconds` we convert it to `_rawFractionalSeconds` and afterwards `_rawFractionalSeconds` is the source of truth.

Reply:
The source of truth is whichever one the user set most recently. That includes whichever one is the one they passed on `init`.

Reply:
Isn't `_fractionalSeconds` just strictly a cache? Or is it any information lost during the conversion?

Reply:
> Isn't `_fractionalSeconds` just strictly a cache of `_rawFractionalSeconds`?

We can say so — no information is lost in the conversion to `_rawFractionalSeconds`.

Reply:
Great, then let's document that. It's otherwise very confusing if you have two variables that contain kinda the same information but not exactly. Ideally we would make `_fractionalSeconds` a computed property but I'm fine with caching this information. If we wouldn't need to support Swift 5.8 we could move to `init` accessors as well to make this even more clear but for now documentation needs to be enough.